torch/nn/functional.py

```diff
@@ -1607,6 +1608,8 @@ def nll_loss(input, target, weight=None, size_average=None, ignore_index=-100,
             out = torch._C._nn.nll_loss2d(
                 input, target, weight, reduction_enum, ignore_index)
             ret = out.view(out_size)
+    else:
+        ret = input  # TODO: remove when jit supports control flow analysis
```

Comment:
@driazati could you make the last elif branch into an else branch and delete this? this code will never be reached